starter-code/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+<meta name="description" content="">
+<meta name="author" content="">
+<title>Welcome to Cluedo!</title>
+
+<!-- Bootstrap core CSS -->
+<link href="vendor/bootstrap/css/bootstrap.min.css" rel="stylesheet">
+<!-- Custom styles for this template -->
+<link href="css/custom.css" rel="stylesheet">
+<link href="https://fonts.googleapis.com/css?family=Homemade+Apple|Permanent+Marker" rel="stylesheet">
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
+</head>
+
+<body>
+
+<!-- Page Content -->
+<div class="container-fluid h-100">
+  <div class="row h-100 justify-content-center align-items-centerh-100 justify-content-center align-items-center">
+    <div class="col-lg-12 text-center mt-4"> <img src="img/header_logo.png" width="150px" alt=""/>
+      <h1 class="mt-5 playgame" onclick="revealMistery(pickMistery());"><a href="#" class="effect-shine">Who murdered Dr. Black?</a></h1>
```

Comment:
You already have a click event function in your JavaScript. You could also write the revealMistery function in there, so your code is more readable and the js and html files are better separated.